change/@itwin-ec3-widget-react-8f39ef3c-42dd-42fd-8683-896a6e977b68.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This will need to be a `minor` version, since now the consumer will need to `initialize` localization. Without calling `initialize` the widget will not work.

Reply:
done